tests/installation/release_notes_from_url.pm

```diff
@@ -19,7 +19,14 @@ sub run {
     enter_cmd "zgrep -oh \"Got release notes.*\" /var/log/YaST2/y2log*";
     assert_screen [qw(got-releasenotes-RPM got-releasenotes-URL)];
     unless (match_has_tag 'got-releasenotes-URL') {
-        record_soft_failure('bsc#1190711 - Release notes source does NOT match expectations or not found in YaST logs, expected source: URL');
+        my $not_public = script_run('grep -r RC /README.BETA');
```

Comment:
where is GMC now?

Reply:
I changed the naming and the logic a bit because indeed it was hard to understand fats what's going on. I hope now it's more clear